src/foam/comics/v2/DAOCreateView.js

```diff
@@ -82,7 +86,13 @@ foam.CLASS({
     {
       name: 'save',
       code: function() {
-        this.config.dao.put(this.data).then((o) => {
+        var clonedData = this.data.clone();
+
+        if ( foam.nanos.auth.LifecycleAware.isInstance(clonedData) ) {
```

Comment:
Can `require` this and change to:
```suggestion
        if ( this.LifecycleAware.isInstance(clonedData) ) {
```
Same for `LifecycleState` below.

Reply:
thanks troykashi